### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by jliberma**

### CODE REVIEW

File: RetrievalAugmentedGeneration/README.md [hunk 1]
```diff
@@ -1,205 +1,361 @@
...
-- Run the following command to build containers.
-    ```
-        source deploy/compose/compose.env;  docker compose -f deploy/compose/docker-compose.yaml build
-    ```
+$ docker ps --format "table {{.ID}}\t{{.Names}}\t{{.Status}}"
+CONTAINER ID   NAMES                     STATUS
+853b74f11e65   llm-playground            Up 15 minutes
+9964f1a797a3   query-router              Up 15 minutes
+e3e3d4871f1a   jupyter-notebook-server   Up 15 minutes
+09150764b74a   triton-inference-server   Up 15 minutes (unhealthy)
```
Comment (shubhadeepd) [inline RetrievalAugmentedGeneration/README.md:221]: This docker name has changed to `llm-inference-server`

**Review by shubhadeepd — CHANGES_REQUESTED** [review]

File: RetrievalAugmentedGeneration/README.md [hunk 2]
```diff
@@ -1,205 +1,361 @@
...
+2285b01dbe0c   milvus-standalone         Up 40 minutes (healthy)
+2c96084539ce   milvus-minio              Up 40 minutes (healthy)
+a839261973b8   milvus-etcd               Up 40 minutes (healthy)
```
[the remark this answers is not in the document]
Comment (jliberma) [reply RetrievalAugmentedGeneration/README.md:224]: I did not deploy it. I do not want it deployed by default. It should be optional. Is it deployed by default now?

File: RetrievalAugmentedGeneration/README.md [hunk 3]
```diff
@@ -1,205 +1,361 @@
...
    4. [Sample frontend](../docs/rag/frontend.md)
-3. [Configuration Guide](../docs/rag/configuration.md): This guide covers different configurations available for this workflow.
-4. [Support Matrix](../docs/rag/support_matrix.md): This covers GPU, CPU, Memory and Storage requirements for deploying this workflow.
+3. Developer RAG [Configuration Guide](../docs/rag/configuration.md)
+4. Developer RAG [Support Matrix](../docs/rag/support_matrix.md)
+
+<hr>
+
+## Enterprise RAG Examples
+
```
[the remark this answers is not in the document]
Comment (jliberma) [reply RetrievalAugmentedGeneration/README.md:361]: its a placeholder but I can delete it, I agree it looks unfinished

**Comment by shubhadeepd** [comment]
Closing this PR, to consolidate changes in a separate commit.

**Pull request closed without merging**